### PR TITLE
add module product_unique_code and move sql_constraints from product_sequence to product_unique_code

### PR DESCRIPTION
**Pull request by chafique-delli**
.

### CODE REVIEW

**Comment by eLBati** [comment]
Thanks.
Some remarks

```
./product_unique_code/__init__.py:5:80: E501 line too long (86 > 79 characters)
./product_unique_code/__init__.py:11:80: E501 line too long (80 > 79 characters)
./product_unique_code/__openerp__.py:5:80: E501 line too long (86 > 79 characters)
./product_unique_code/__openerp__.py:11:80: E501 line too long (80 > 79 characters)
./product_unique_code/__openerp__.py:26:11: E203 whitespace before ':'
./product_unique_code/__openerp__.py:27:14: E203 whitespace before ':'
./product_unique_code/__openerp__.py:30:14: E203 whitespace before ':'
./product_unique_code/__openerp__.py:31:15: E203 whitespace before ':'
./product_unique_code/__openerp__.py:33:80: E501 line too long (85 > 79 characters)
./product_unique_code/__openerp__.py:35:14: E203 whitespace before ':'
./product_unique_code/__openerp__.py:38:11: E203 whitespace before ':'
./product_unique_code/product.py:5:80: E501 line too long (86 > 79 characters)
./product_unique_code/product.py:24:1: F401 'fields' imported but unused
./product_unique_code/product.py:25:1: F401 '_' imported but unused
./product_unique_code/product.py:32:5: E303 too many blank lines (2)
```

File: product_unique_code/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+#
+##############################################################################
+
+{
+    'name' : 'Product Unique Code',
+    'version' : '0.1',
+    "author": "Akretion",
+    "website": "http://www.akretion.com",
+    "license" : "AGPL-3",
+    "category" : "Generic Modules/Inventory Control",
```
Comment (LeartS) [inline product_unique_code/__openerp__.py]: Is this the correct category?

**Comment by chafique-delli** [comment]
thinks to merge

**Comment by eLBati** [comment]
@chafique-delli please rebase

**Comment by pedrobaeza** [comment]
Is this still valid?

**Comment by pedrobaeza** [comment]
As there's no answer, I close. Feel free to reopen in case there's interest.

**Pull request closed without merging**